_rules/html-page-lang-xml-lang-match-5b7ae0.md

```diff
@@ -50,52 +53,36 @@ Since most assistive technologies will consistently use `lang` over `xml:lang` w
 
 #### Passed Example 1
 
-`html` element with matching value for `lang` and `xml:lang`.
+`html` element with matching primary language subtags for `lang` and `xml:lang`.
 
 ```html
 <html lang="en" xml:lang="en"></html>
 ```
 
 #### Passed Example 2
 
-`html` element with varied case but matching value for `lang` and `xml:lang`.
+`html` element with matching primary and extended language subtags for `lang` and `xml:lang`.
 
 ```html
-<html lang="en" xml:lang="En"></html>
+<html lang="en-GB" xml:lang="en-GB"></html>
 ```
 
-#### Passed Example 3
-
-`html` element with varied values but matching primary sub-tag value for `lang` and `xml:lang`.
-
-```html
-<html lang="en" xml:lang="en-GB"></html>
-```
-
-#### Passed Example 4
-
-`html` element with varied values but matching primary sub-tag value for `lang` and `xml:lang`.
-
-```html
-<html lang="en-GB" xml:lang="en"></html>
-```
+### Failed
 
-#### Passed Example 5
+#### Failed Example 1
 
-`html` element with varied values but matching primary sub-tag value for `lang` and `xml:lang`, albeit the value `XYZ` is not valid.
+`html` element with non-matching primary language subtags for `lang` and `xml:lang`.
 
 ```html
-<html lang="en-XYZ" xml:lang="en"></html>
+<html lang="fr" xml:lang="en"></html>
 ```
 
-### Failed
-
-#### Failed Example 1
+#### Failed Example 2
 
-`html` element with non-matching value for `lang` and `xml:lang`.
+`html` element with non-matching extended language subtags for `lang` and `xml:lang`. While the primary language subtag is Chinese for both the `lang` and `xml:lang` attribute, the extended language subtag is Cantonese for the `lang` attribute and Mandarin for the `xml:lang` attribute.
 
 ```html
-<html lang="fr" xml:lang="en"></html>
+<html lang="zh-yue" xml:lang="zh-cmn"></html>
 ```
 
 ### Inapplicable
```

Comment:
We're missing inapplicable examples with empty or whitespace only `lang` attributes

You might also want to add another example with a `<math>` element